apps/assets/src/components/topBarButtons/RequestButton.tsx

```diff
@@ -1,25 +1,28 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/apps/blob/main/LICENSE)
+
 import { StoreType } from "evmos-wallet";
 import { RequestIcon } from "icons";
 import { useSelector } from "react-redux";
 
-import {  PrimaryButton } from "ui-helpers";
+import { PrimaryButton } from "ui-helpers";
 
 export const RequestButton = () => {
-    const handleOnClick = () => {
-        //  TODO: add logic
-    }
+  const handleOnClick = () => {
+    //  TODO: add logic
+  };
+
+  const wallet = useSelector((state: StoreType) => state.wallet.value);
```

Comment:
you could  use the useAccount here to check if it's active
```suggestion
     const { isConnected } = useAccount();
```